indent/vue.vim

```diff
@@ -34,31 +41,31 @@ let s:html_indent = s:get_indentexpr('html')
 
 let b:did_indent = 1
 
-setlocal indentexpr=GetVueIndent()
+setlocal indentexpr=GetVueIndent(v:lnum)
 
 if exists('*GetVueIndent')
   finish
 endif
 
-function! GetVueIndent()
+function! GetVueIndent(lnum)
   for language in s:languages
     let opening_tag_line = searchpair(language.pairs[0], '', language.pairs[1], 'bWr')
 
     if opening_tag_line
-      execute 'let indent = ' . get(language, 'indentexpr', -1)
+      let indent = language.indentexpr
```

Comment:
`get()` was being used here to return `-1` (which means "do not change the indentation") when the indent file for that language is not present.

Reply:
Honestly I couldn't tell that this actually affected anything when I changed it-- tested it with missing indent files